specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/examples/ManagedHsm_putPrivateEndpointConnection.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "sample-group",
+    "name": "sample-mhsm",
+    "privateEndpointConnectionName": "sample-pec",
+    "api-version": "2021-04-01-preview",
+    "properties": {
+      "properties": {
+        "privateLinkServiceConnectionState": {
+          "status": "Approved",
+          "description": "My name is Joe and I'm approving this."
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sample-group/providers/Microsoft.KeyVault/managedhsms/sample-mhsm/privateEndpointConnections/sample-pec",
+        "name": "sample-pec",
+        "type": "Microsoft.KeyVault/managedhsms/privateEndpointConnections",
+        "properties": {
+          "provisioningState": "Succeeded",
```

Comment:
Given there's a provisioning state, why is this not a long-running operation (LRO)? I see that deleting a private endpoint is. I mean, if fast enough, great! Just seemed odd enough to call out.

Reply:
This is asynchronous operation as defined in spec file.